<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camerasetup</class>
 <widget class="QWidget" name="camerasetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">outline: none;
color: rgb(255, 255, 255);
background-color: rgb(62, 63, 63);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="SerialNumber">
      <property name="styleSheet">
       <string notr="true">color:white; font-size:15px</string>
      </property>
      <property name="text">
       <string>SerialNumber:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serialEditor">
      <property name="styleSheet">
       <string notr="true">min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="CameraName">
      <property name="styleSheet">
       <string notr="true">color:white; font-size:15px</string>
      </property>
      <property name="text">
       <string>CameraName:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="camEditor">
      <property name="styleSheet">
       <string notr="true">min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ImageStyleEditor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:15px</string>
   </property>
   <property name="text">
    <string>ImageStyle:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="RGB_check">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 15px;
    color: white;
    font-size: 15px;
}

QCheckBox::indicator {
    width: 17px;
    height: 15px;
    border: 2px solid gray;
    border-radius: 5px;
    border-color: rgb(35, 143, 159);
    background-color: rgb(50, 50, 63);
}

QCheckBox::indicator:checked {
    background-color: rgb(0, 255, 0);
}
</string>
      </property>
      <property name="text">
       <string>RGB</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="DEPTH_check">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 15px;
    color: white;
    font-size: 15px;
}

QCheckBox::indicator {
    width: 17px;
    height: 15px;
    border: 2px solid gray;
    border-radius: 5px;
    border-color: rgb(35, 143, 159);
    background-color: rgb(50, 50, 63);
}

QCheckBox::indicator:checked {
    background-color: rgb(0, 255, 0);  
}
</string>
      </property>
      <property name="text">
       <string>DEPTH</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="POINTCLOUD_check">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 15px;
    color: white;
    font-size: 15px;
}

QCheckBox::indicator {
    width: 17px;
    height: 15px;
    border: 2px solid gray;
    border-radius: 5px;
    border-color: rgb(35, 143, 159);
    background-color: rgb(50, 50, 63);
}

QCheckBox::indicator:checked {
    background-color: rgb(0, 255, 0);  
}
</string>
      </property>
      <property name="text">
       <string>POINTCLOUD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
